Src/Support/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -115,8 +115,13 @@ public new class Initializer : AuthorizationCodeFlow.Initializer
             /// <see cref="Google.Apis.Auth.OAuth2.GoogleAuthConsts.AuthorizationUrl"/>, and Token server URL to 
             /// <see cref="Google.Apis.Auth.OAuth2.GoogleAuthConsts.TokenUrl"/>.
             /// </summary>
-            public Initializer()
-                : base(GoogleAuthConsts.AuthorizationUrl, GoogleAuthConsts.TokenUrl)
+            public Initializer() : this(GoogleAuthConsts.AuthorizationUrl, GoogleAuthConsts.TokenUrl) { }
+
+            /// <summary>Constructs a new initializer.</summary>

```

Comment:
I pushed too early, yes it now takes a revoke URL too. I'll add an explanation of why this is here.